pennylane/drawer/drawable_layers.py

```diff
@@ -94,6 +94,12 @@ def drawable_layers(ops, wire_map=None):
     measured_wires = {}
     measured_layers = {}
 
+    # Collect all mid-circuit measurements used for classical conditioning
+    cond_measurements = set()
+    for op in ops:
+        if op.__class__.__name__ == "Conditional":
```

Comment:
```suggestion
        if op.name == "Conditional":
```
Or an `isinstance` check?

Reply:
Can't use `isinstance` because of circular imports

Reply:
ugk. circular imports grumble grumble.